libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -283,7 +283,13 @@ func (m *Manager) Apply(pid int) error {
 	}
 
 	if _, err := theConn.StartTransientUnit(unitName, "replace", properties, nil); err != nil {
-		return err
+		if dbusError, ok := err.(dbus.Error); ok {
```

Comment:
The logic is a little confusing.  It would be better to have this as a function and reduce a few ifs.

```go
if _, err := theConn.StartTransientUnit(unitName, "replace", properties, nil); err != nil && !isUnitExist(err) {
    return err
}
```